pkg/propertiesformatter/propertyFormatter.go

```diff
@@ -21,27 +22,30 @@ import (
 // PropertyFormatter - implementations can take a collection of properties results
 // and turn them into a string for display to the user.
 const (
+	//properties display
 	HEADER_PROPERTY_NAMESPACE = "namespace"
 	HEADER_PROPERTY_NAME      = "name"
 	HEADER_PROPERTY_VALUE     = "value"
-)
-
-type FormattableProperty struct {
-	Namespace string
-	Name      string
-	Value     string
-}
 
-func NewFormattableProperty() FormattableProperty {
-	this := FormattableProperty{}
-	return this
-}
+	//namespaces display
+	HEADER_NAMESPACE          = "namespace"
+	HEADER_NAMESPACE_TYPE     = "type"
+	HEADER_NAMESPACE_URL      = "url"
+)
 
 type PropertyFormatter interface {
-	FormatProperties(propertyResults []FormattableProperty) (string, error)
+	FormatProperties(propertyResults []galasaapi.CpsProperty) (string, error)
+	FormatNamespaces(namespaces []galasaapi.Namespace) (string, error)
 	GetName() string
 }
 
+func GetNameAndNamespace(fullName string) (string, string) {
+	splitName := strings.SplitN(fullName, ".", 2)
+	namespace := splitName[0]
+	name := splitName[1]
```

Comment:
Could this throw an out of bounds error?

Reply:
This change was merged from main so not sure why it is showing as an addition. Property names that come in will have at least one dot in it so this shouldn't be an issue.